lib/radiator/directory/episode.ex

```diff
@@ -143,6 +143,18 @@ defmodule Radiator.Directory.Episode do
     end
   end
 
+  def generate_short_id(short_id_base, number) when is_integer(number) do
+    generate_short_id(short_id_base, to_string(number))
+  end
+
+  def generate_short_id(short_id_base, number) when is_binary(number) do
+    "#{short_id_base}#{String.pad_leading(number, 3, "0")}"
+  end
+
+  def generate_short_id(short_id_base, _) do
+    generate_short_id("#{short_id_base}_t", :rand.uniform(1000))
```

Comment:
What's the reasoning here to generate a random short id instead of leaving it blank?

Reply:
Not much, more the idea if we want short_ids to provide some independent from the fact if the imported one had episode numbers. And since we can't reliably deduce episode numbers going random was just were I got to.